Remotion/Core/Development.Analyzers/RMCORE0001_SafeContextAnalyzer.cs

```diff
@@ -17,15 +17,15 @@ private record SymbolContext (ITypeSymbol? SafeContextBoundarySymbol);
     public static readonly DiagnosticDescriptor AlternativeDescriptor = new DiagnosticDescriptor(
         "RMCORE0001",
         "Use SafeContext instead of typical API",
-        "'{0}' should not be used with SafeContext, use 'Remotion.Context.SafeContext.{1}' instead.",
+        "'{0}' should not be used with SafeContext, use 'Remotion.Context.SafeContext.{1}' instead",
```

Comment:
Micorsoft now uses a form without a trailing period in the analyzers? it would be best to have this as a separate commit with an explainatory commit message.

Context: I remember a discussion (or change) about this from 12-24 months ago.  I would like to understand when and how this was changed (or if we always did it "wrong").
cc @a-ctor & @Frazerus

Reply:
This has always been with `.` at the end here, I do not remember a discussion about this however.

Reply:
I just found a current example and yeah, Microsoft doesn't place a `.` at the end. It probably dates back to Marcel. 
Question for @a-ctor  and @Frazerus change this now, later or not at all?
Question for @KuJo7 : is there a ready-made assertion in the Microsoft-Library which forced this change or did you simply notice it and adapt it to be consistent with Microsoft?

Reply:
With a newer version of roslyn a check for `<EnforceExtendedAnalyzerRules>true</EnforceExtendedAnalyzerRules>` was introduced and this is what causes all those changes now as it has an error severity. I would do the changes now as they don't seem to be too much work.

Reply:
See note in Core.Development.Analyzers.csproj